mmv1/products/lustre/Instance.yaml

```diff
@@ -0,0 +1,127 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: Instance
+description: Description
+base_url: projects/{{project}}/locations/{{location}}/instances
+update_mask: true
+self_link: projects/{{project}}/locations/{{location}}/instances/{{instance_id}}
+create_url: projects/{{project}}/locations/{{location}}/instances?instanceId={{instance_id}}
+update_verb: PATCH
+id_format: projects/{{project}}/locations/{{location}}/instances/{{instance_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/instances/{{instance_id}}
+examples:
+  - name: lustre_instance_basic
+    primary_resource_id: 'instance'
+    vars:
+      name: 'instance'
```

Comment:
The ids here should each include a hyphen, which signals to our generator that we want the value used in tests to have a sweepable prefix and a randomize suffix. So for example, name could be `my-instance` or similar. I think the same applies to network_name and address_name.

Reply:
For network name this actually caused a lot of issue as for Lustre the min and max capacity is tied to the network name. For this reason I decided to remove the network_name.